tweets.py

```diff
@@ -9,8 +9,9 @@
 import smtplib, ssl
 from email.mime.text import MIMEText
 
-CONFIG_FILE="./config.json"
+CONFIG_FILE="./twitter_data_config.json"
 LOG_LEVEL="WARN"
+USE_MAIL=False
 
 def mike_bs():
     # mail stuff
```

Comment:
since you moved this to a config, you should remove your personal info from this file. that was the whole point right?